avro2py/codegen.py

```diff
@@ -363,10 +362,16 @@ def bridge_namespaces(from_namespace: List[str], to_namespace: List[str], target
         remaining_to_components = [c for c in paired_remaining_to_components if c is not None]
 
         if remaining_from_components:
+            level = len(remaining_from_components)
+            for namespace_name in self.namespaces.keys():
+                namespace_parts = namespace_name.split('.')
+                if namespace_parts[0:len(from_namespace)] == from_namespace and len(namespace_parts) > len(from_namespace):
```

Comment:
This condition is hard to follow. Would you mind breaking it up like e.g.:
```suggestion
                namespace_is_in_from_namespace = all(
                    namespace_component == from_namespace_component
                    for namespace_component, from_namespace_component in zip(namespace_parts, from_namespace)
                )
                namespace_is_deeper_than_from_namespace = len(namespace_parts) > len(from_namespace_parts)
                namespace_is_child_of_from_namespace = namespace_is_in_from_namespace and namespace_is_deeper_than_from_namespace 
                if namespace_is_child_of_from_namespace:
```

Reply:
Alternatively, if you just did a check like `if from_namespace == namespace_parts: continue` before the `zip` check, I think the `zip` would be sufficient - you're just trying to see if the `from_namespace` has _any_ children, and need to make sure that you're excluding the trivial case of comparing `from_namespace` to itself, right?

Reply:
I used zip for this in the replacement version.